docs/HISTORY.txt

```diff
@@ -1,10 +1,14 @@
 Changelog
 =========
 
-trunk (unreleased)
-------------------
+0.7 (unreleased)
```

Comment:
I think we can update the date here and add a release in GitHub

Reply:
done in https://github.com/davilima6/plonetheme.motion/releases/tag/0.7

thanks!!